app/models/spree/payment/processing.rb

```diff
@@ -39,16 +39,18 @@ def capture!
         protect_from_connection_error do
           check_environment
 
-          if payment_method.payment_profiles_supported?
-            # Gateways supporting payment profiles will need access to credit card object because this stores the payment profile information
-            # so supply the authorization itself as well as the credit card, rather than just the authorization code
-            response = payment_method.capture(self, source, gateway_options)
-          else
-            # Standard ActiveMerchant capture usage
-            response = payment_method.capture(money.money.cents,
+          response = if payment_method.payment_profiles_supported?
```

Comment:
I dont like this rubocop rule, am I the only one? I always find the result to look worse than before and the solution is always to extract the if clause to a method. I am not requesting any change, just a comment.

Reply:
I don't have a strong opinion on this. I do find it a bit more readable cause otherwise the assignment is easy to miss. As many things in Ruby though, abusing it by having a long if/else beats readability anyway, so extracting a method in those case makes sense.